### PR TITLE
Refactor/map screen UI

### LINKED ISSUE #400: Improve map screen based on the figma

### PR DESCRIPTION
**Pull request by elvan03**
In this PR,  I update the UI of the map screen. 

1) I have customised the map to look darker than the default. This matches better with the style of our app + makes pins(markers) more visible. The chosen colours for the map are available in the map_style.json file.

2) I have added a file (MapMarkerUtils) that allowed me to customise the pins(markers) on the map. Now, the markers have a colour that is defined by the category they belong to, for example, green if it's a Sport etc...

I have added a test file and have > 80% in coverage on new code.

closes #400

### CODE REVIEW

File: app/src/main/java/com/android/sample/ui/map/MapMarkerUtils.kt
Comment (Yassyass181104) [inline app/src/main/java/com/android/sample/ui/map/MapMarkerUtils.kt:1]: This implementation of our own pin looks correct and very simple to use

File: app/src/main/java/com/android/sample/ui/map/Map.kt
Comment (Yassyass181104) [inline app/src/main/java/com/android/sample/ui/map/Map.kt:1]: You use the new pin implementation to have them colored depending on the category, very good.

**Review by Yassyass181104 — CHANGES_REQUESTED** [review]
Overrall the implementation looks very good but the only problem I can point is that the tests you have created must be moved to the unit tests folder but they seem to be complete

**Comment by elvan03** [comment]
Thanks for your review Yassine! The test file is already in the unit tests folder :))

**Review by Yassyass181104 — APPROVED** [review]

**Comment by Yassyass181104** [comment]
> Thanks for your review Yassine! The test file is already in the unit tests folder :))

My bad, thanks for the clarification

**Pull request merged**